src/Test/MailTestCaseTrait.php

```diff
@@ -84,11 +87,26 @@ public function assertMailSentForRecipients(array $expectedEmails, string $mailC
         }
         $expectedEmails = \array_filter($expectedEmails);
 
-        $this->assertCount(0, \sprintf(
+        self::assertCount(0, $expectedEmails, \sprintf(
             'No mail%s was sent for "%s" among %d mail(s).',
             $mailClass ? " ($mailClass)" : '',
             \implode('", "', $expectedEmails),
-            $mailClass ? $mailPost->getMailsCountForClass($mailClass) : $mailPost->getMailsCount()
+            $mailClass ? $mailPost->countMailsForClass($mailClass) : $mailPost->countMails()
         ));
     }
+
+    public static function assertMessageRecipient(string $expectedEmail, string $expectedName, array $expectedVars, RecipientInterface $recipient): void
+    {
+        self::assertSame($expectedEmail, $recipient->getEmail());
+        self::assertSame($expectedName, $recipient->getName());
+        self::assertSame($expectedVars, $recipient->getTemplateVars());
+    }
+
+    protected function clearMails(string $mailPostName = null): void
+    {
+        if (!$this->mailPost) {
+            $this->mailPost = $this->getMailPost($mailPostName);
+        }
+        $this->mailPost->clearMails();
```

Comment:
Can you just add a new line after a `if` statement please ?

Reply:
done